builtin/providers/aws/resource_aws_spot_fleet_request.go

```diff
@@ -564,8 +583,28 @@ func resourceAwsSpotFleetRequestCreate(d *schema.ResourceData, meta interface{})
 		DryRun:                 aws.Bool(false),
 	}
 
-	log.Printf("[DEBUG] Requesting spot fleet with these opts: %+v", spotFleetOpts)
-	resp, err := conn.RequestSpotFleet(spotFleetOpts)
+	fmt.Printf("[DEBUG] Requesting spot fleet with these opts: %+v", spotFleetOpts)
```

Comment:
Ah, you're right. The output didn't show up in the tests until I changed this but I think it's incorrect (I was probably missing the TF_LOG envvar).